packages/amplication-client/src/Purchase/PurchasePage.tsx

```diff
@@ -56,6 +56,21 @@ const selectedPlanAction = {
   },
 };
 
+const getPlanPrice = (
+  selectedBillingPeriod: BillingPeriod,
+  pricePoints: Price[]
+) => {
+  const UNKNOWN = "unknown";
+  if (!pricePoints.length) return UNKNOWN;
+
+  return pricePoints.reduce((price: string, pricePoint: Price) => {
+    if (pricePoint.billingPeriod === selectedBillingPeriod)
+      price = `${pricePoint.amount}${pricePoint.currency}`;
+
+    return price;
+  }, UNKNOWN);
```

Comment:
I think this data is not relevant and also not correct. 
It will return the list price of the selected plan and not the real price of the current plan.
@GreenMachine01  I think we can remove it. WDYT?

Reply:
this was the feature request